ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -224,6 +233,15 @@ static boolean bug1162183Affected(Map<String, String> attributes) {
     return ParseUri.TELEMETRY.equals(attributes.get(Attribute.DOCUMENT_NAMESPACE))
         && affectedDocumentTypes.contains(attributes.get(Attribute.DOCUMENT_TYPE))
         && attributes.get(Attribute.APP_VERSION) != null
-        && attributes.get(Attribute.APP_VERSION).matches("([0-3][0-9]|4[0-1])\\..*"); // 41 or older
+        && attributes.get(Attribute.APP_VERSION).matches("^([0-3][0-9]|4[0-1])\\..*"); // >= 41
+  }
+
+  // See bug 1642386 for discussion of affected versions, etc.
+  @VisibleForTesting
+  static boolean bug1642386Affected(Map<String, String> attributes) {
+    return ParseUri.TELEMETRY.equals(attributes.get(Attribute.DOCUMENT_NAMESPACE))
+        && attributes.get(Attribute.DOCUMENT_TYPE).equals("sync")
+        && attributes.get(Attribute.APP_VERSION) != null
+        && attributes.get(Attribute.APP_VERSION).matches("^(25|26)\\..*"); // 25 or 26
```

Comment:
We may want to change so that it catches all versions less than 26, based on the results from https://sql.telemetry.mozilla.org/queries/71851/source#180308. 22, 23, and 24 are the next highest sources of errors. This will take us most of the way there though.

Reply:
Ah I see. That seems reasonable, I'll open a separate PR for applying this to earlier versions too